widget/entry_test.go

```diff
@@ -180,7 +179,7 @@ func TestEntry_OnKeyDown_Insert(t *testing.T) {
 }
 
 func TestEntry_OnKeyDown_Newline(t *testing.T) {
-	entry := &Entry{MultiLine: true}
```

Comment:
If this change is needed then we have broken something - can it be reverted?

Reply:
See comment below